### PR TITLE
Replace repository with new content

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by Ono-Saito** [comment]
Replaced the content

**Comment by Ono-Saito** [comment]
ok

File: app/features/conference/components/Conference.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ import { conferenceEnded, conferenceJoined } from '../actions';
 import JitsiMeetExternalAPI from '../external_api';
 import { LoadingIndicator, Wrapper } from '../styled';
 
-const ENABLE_REMOTE_CONTROL = false;
```
Comment (Ono-Saito) [inline app/features/conference/components/Conference.js:19]: This enables remote control

File: app/features/config/index.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ export default {
     /**
      * Application name.
      */
-    appName: 'Jitsi Meet',
```
Comment (Ono-Saito) [inline app/features/config/index.js:16]: Changed the name for Aisle School.

File: app/i18n/index.js [hunk 3]
```diff
@@ -8,10 +8,10 @@ const languages = {
     es: { translation: require('./lang/es.json') },
     fr: { translation: require('./lang/fr.json') },
     gl: { translation: require('./lang/gl.json') },
-    hi: { translation: require('./lang/hi.json') },
     hr: { translation: require('./lang/hr.json') },
     hu: { translation: require('./lang/hu.json') },
     it: { translation: require('./lang/it.json') },
+    ja: { translation: require('./lang/ja.json') },
```
Comment (Ono-Saito) [inline app/i18n/index.js:14]: Added Japanese.

File: app/i18n/index.js [hunk 4]
```diff
@@ -20,8 +20,8 @@ const languages = {
     'zh-TW': { translation: require('./lang/zh-TW.json') }
 };
 
-const detectedLocale = navigator.language;
-
+// const detectedLocale = navigator.language;
+const detectedLocale = 'ja';
```
Comment (Ono-Saito) [inline app/i18n/index.js:24]: Set the default language Japanese.

**Comment by saghul** [comment]
Please stop sending bogus PRs.

**Comment by Ono-Saito** [comment]
I'm sorry but it is an error. I only wanted to merge to my repository. But
it was forked from you repository. So it occurred.
Please understand me.

**Pull request closed without merging**